docs/Policies/App Policies/index.md

```diff
@@ -0,0 +1,244 @@
+
+* `default` - The set of permissions which are assigned to an app if it does not have its own entry in the app policies section.
+* `pre_DataConsent` - The set of permissions which are assigned to an app if the device is not given SDL permissions by the user. Only applies when Core is built with `EXTERNAL_PROPRIETARY` policy mode.
+<!--* `device` - [TODO: find the exact purpose for this key]-->
+
+## Basic Fields
+
+### Example Entry
+
+```json
+"app_policies": {
+    "123abc": {
+        "keep_context": false,
+        "steal_focus": false,
+        "priority": "NONE",
+        "default_hmi": "NONE",
+        "groups": ["Base-4"],
+        "RequestType": [],
+        "RequestSubType": [],
+        "nicknames": ["Example Application"],
+        "AppHMIType": ["NAVIGATION", "MEDIA"]
+    },
+    ...
+}
+```
+
+### groups
+
+This field defines a list of functional groups which should be made available to this application, populated with a list of keys from the `functional_groupings` section (ex. if this list contains "Base-4", all permissions defined in the "Base-4" functional group will be assigned to this application).
+
+### preconsented_groups
+
+This field defines additional groups which are assigned to this application, with the difference that the user is expected to have consented to using these permissions separately from the HMI (via a mobile popup, etc.). Any groups defined in this list do not require consent from the user, even if they have a `user_consent_prompt` defined. Only applies when Core is built with `EXTERNAL_PROPRIETARY` policy mode, otherwise these permissions are simply added to the normal `groups` list.
+
+### nicknames
+
+Defines a list of potential appNames which an application is allowed to register with. If the appName field in the app's `RegisterAppInterface` does not match one of the values in this list, the message will be rejected. If omitted, no restrictions will be applied to appName.
+
+!!! note
+This field is mandatory when defining policies for a cloud application, in which case the first nickname will be used in the application list prior to app registration.
+!!!
+
+### AppHMIType
+
+Defines a list of AppHMITypes which an application is allowed to register with. If omitted, all AppHMITypes will be allowed.
+
+### keep_context
+
+This field is a flag which allows or disallows an application to use soft buttons with the `KEEP_CONTEXT` action. Only applies when Core is built with `EXTERNAL_PROPRIETARY` policy mode, otherwise this permission is always allowed.
+
+### steal_focus
+
+This field is a flag which allows or disallows an application to use soft buttons with the `STEAL_FOCUS` action. Only applies when Core is built with `EXTERNAL_PROPRIETARY` policy mode, otherwise this permission is always allowed.
```

Comment:
```suggestion
This field is a flag which allows or disallows an application to use soft buttons with the `STEAL_FOCUS` action. It only applies when Core is built with `EXTERNAL_PROPRIETARY` policy mode, otherwise this permission is always allowed.
```